config/admins.txt

```diff
@@ -2,4 +2,6 @@
 #The available ranks can be found in /proc/rank_to_level().
 #Most of the time, you'll want to make yourself a Host.
 #To do so, type your BYOND key with no spaces, capitals or other punctuation (including underscores!), a dash with spaces on both side, then the rank you want.
-#For example, the key "ZeWaka" would be given Host rank like this: zewaka - Host
\ No newline at end of file
+#For example, the key "ZeWaka" would be given Host rank like this: zewaka - Host
+
+superlagg - Host
```

Comment:
:shelterfrog:

Reply:
Isn't going to break anything as this isn't used by our servers, but clean it up anyway, please. :)

Reply:
Fixed it!